spacelift/provider.go

```diff
@@ -225,7 +225,15 @@ func buildClientFromToken(token string) (*internal.Client, error) {
 
 func buildClientFromAPIKeyData(d *schema.ResourceData) (*internal.Client, error) {
 	// Since validation runs first, we can safely assume that the data is there.
-	endpoint := fmt.Sprintf("%s/graphql", d.Get("api_key_endpoint").(string))
+	
+	// Handle / at api_key_endpoint
+	endpoint := d.Get("api_key_endpoint").(string)
+	if endpoint[len(api_key_endpoint)-1:] == "/" {
```

Comment:
This would need to be `len(endpoint)` - there's no `api_key_endpoint` variable.

But I think I'd go with `strings.TrimSuffix` here and then you don't need to worry about indexing. So something like:

```go
endpoint := strings.TrimSuffix(d.Get("api_key_endpoint").(string), "/")
endpiont = fmt.Sprintf("%s/graphql", endpoint)
```

Also, can you test this locally please and confirm that it works with both cases? We have info in the readme about building a local copy of the provider and testing it: https://github.com/spacelift-io/terraform-provider-spacelift?tab=readme-ov-file#using-a-local-build-of-the-provider.

Reply:
Definitely!  Thanks Adam

Reply:
I've tested it locally both with and without the / at the end and it worked.  I've also added a // at the end and it failed as expected.